{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "6e308da1_efc68320",
        "filename": "cue/testdata/cycle/self.txtar",
        "patchSetId": 1
      },
      "lineNbr": 300,
      "author": {
        "id": 1017723
      },
      "writtenOn": "2022-11-30T13:02:18Z",
      "side": 0,
      "message": "The loss of this error seems to be a net negative, no? i.e. we have dropped the error that referred to the index position, leaving only the reference to the entire list. Highlighted in this case by the fact that we have two errors referring to list.error2.a:\n\n    list.error2.a: conflicting values \"3\" and \"1\":\n    list.error2.a: conflicting values \"3\" and \"2\":\n    \nWas this intentional?",
      "revId": "27155e9be7c9b4304542432b50fb123dc3b03a4e",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    }
  ]
}